src/coreclr/ToolBox/superpmi/superpmi-shared/spmirecordhelper.h

```diff
@@ -194,13 +208,16 @@ inline void SpmiRecordsHelper::Restore_CORINFO_RESOLVED_TOKENout(
     Agnostic_CORINFO_RESOLVED_TOKENout& tokenOut,
     LightWeightMap<key, value>* buffers)
 {
-    pResolvedToken->hClass       = (CORINFO_CLASS_HANDLE)tokenOut.hClass;
-    pResolvedToken->hMethod      = (CORINFO_METHOD_HANDLE)tokenOut.hMethod;
-    pResolvedToken->hField       = (CORINFO_FIELD_HANDLE)tokenOut.hField;
-    pResolvedToken->pTypeSpec    = (PCCOR_SIGNATURE)buffers->GetBuffer(tokenOut.pTypeSpec_Index);
-    pResolvedToken->cbTypeSpec   = (ULONG)tokenOut.cbTypeSpec;
-    pResolvedToken->pMethodSpec  = (PCCOR_SIGNATURE)buffers->GetBuffer(tokenOut.pMethodSpec_Index);
-    pResolvedToken->cbMethodSpec = (ULONG)tokenOut.cbMethodSpec;
+    if (pResolvedToken != nullptr)
```

Comment:
This is only called for resolveToken and tryResolveToken. Can pResolvedToken ever be null for those?

Reply:
Probably not? I can try to minimize where we do this, if you think it's important.

Reply:
I guess it's not that important.